src/test/resources/tweetStatus.txt

```diff
@@ -0,0 +1,6 @@
+ {
```

Comment:
should the file be with `.json` extension?

Reply:
So, this is a string saved in Json format in a file. Technically there is no issue with .txt extension. But are you suggesting .json extension to imply things better ?

Reply:
yep.